go/vt/vtgate/schema/tracker_test.go

```diff
@@ -194,7 +194,7 @@ func TestTrackerNoLock(t *testing.T) {
 	for i := 0; i < 500000; i++ {
 		select {
 		case ch <- th:
-		case <-time.After(5 * time.Millisecond):
+		case <-time.After(10 * time.Millisecond):
```

Comment:
No guarantee that this will not be flaky even then. We should look at the new synctest package in 1.24 to see if that can be used to reduce flakiness in tests that depend on time. We can approve and merge for now.

https://go.dev/blog/synctest

Reply:
I can look into for `main`. This needs to be ported to v20 and v21.